opentelemetry/allowed-external-types.toml

```diff
@@ -0,0 +1,19 @@
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+# This is used with cargo-check-external-types to reduce the surface area of downstream crates from
+# the public API. Ideally this can have a few exceptions as possible.
+allowed_external_types = [
+    "indexmap::map::core::Entry",
```

Comment:
i hope we can get to zero for opentelemtry api!

Reply:
Agree, that would be an ideal scenario. In Otel-C++, there is no external dependency for API + SDK + Memory Exporter + OStream Exporter.